1.1/spec/01-Preamble.adoc

```diff
@@ -1,65 +1,80 @@
-== Preface
-This standard defines a set of SPARQL extension functions [W3C SPARQL], a set of RIF rules [W3C RIF Core], and a core RDF/OWL [12,14,16] vocabulary for geographic information based on the General Feature Model <<ISO19109>>, Simple Features [ISO 19125-1], Feature Geometry <<ISO19107>> and SQL MM <<ISO13249>>.
+== i.    Preface
+The GeoSPARQL standard defines:
 
-== Submitting organizations
-The following organizations submitted this Implementation Specification to the Open Geospatial Consortium Inc.:
-
-a) Australian Bureau of Meteorology
-
-b) Bentley Systems, Inc.
-
-c) CSIRO
-
-d) Defence Geospatial Information Working Group (DGIWG)
-
-e) GeoConnections - Natural Resources Canada
-
-f) Interactive Instruments GmbH
-
-g) Oracle America
+* a formal _profile_
+* **this specification document**
+** a core RDF/OWL ontology for geographic information representation
+* a set of SPARQL extension functions
+* a Functions & Rules vocabulary, derived from the ontology
+* a Simple Features feature types vocabulary
+* a set of RIF rules, and
+* SHACL shapes for RDF data validation
 
-h) Ordnance Survey
-
-i) Raytheon Company
-
-j) Traverse Technologies, Inc.
-
-k) US Geological Survey (USGS)
+This document has the role of _specification_ and authoratitivly defines many of the standard's elements, including the ontology, SPRQL functions and function and rule vocabularies. Complete descriptions of the standard's parts and their roles are given in the Introduction in the section <<GeoSPARQL Standard structure>>.
 
+== ii.   Submitting organizations
+The following organizations submitted this Implementation Specification to the Open Geospatial Consortium Inc.:
 
-== Submission contact points
+[loweralpha]
+.. Australian Bureau of Meteorology
+.. Bentley Systems, Inc.
+.. CSIRO
+.. Defence Geospatial Information Working Group (DGIWG)
+.. GeoConnections - Natural Resources Canada
+.. Interactive Instruments GmbH
+.. GeoScape Australia
+.. Mainz University Of Applied Sciences
+.. Oracle America
+.. Ordnance Survey
+.. Raytheon Company
+.. SURROUND Australia Pty Ltd.
+.. Traverse Technologies, Inc.
+.. US Geological Survey (USGS)
+
+
+=== Submission contact points
```

Comment:
Should use iii. section number here

Reply:
Fixed in https://github.com/opengeospatial/ogc-geosparql/pull/102/commits/3b128f7cc36e7e2db354395b399e7daca2cc7eb0